glossary/project.md

```diff
@@ -1 +1,3 @@
-A distinct thing we are developing as part of a [product](product.md). [Achievables](achievable.md) are often part of a project.		
+A distinct thing we are developing as part of a [product](product.md). [Achievables](achievable.md) are often part of a project.
+
+**As of August 19, 2016, projects have been replaced by [milestones](milestone.md).** 
```

Comment:
Is this necessary? Maybe I would recommend deleting  the whole file project.md from the guides to prevent confusion.